.github/workflows/merge-pr-to-preview.yml

```diff
@@ -7,7 +7,9 @@ on:
       - master
 jobs:
   sync-pull-request:
-    if: contains(github.event.pull_request.labels.*.name, 'preview')
+    if: |
+      contains(github.event.pull_request.labels.*.name, 'preview') &&
+      github.repository_owner == 'betterscientificsoftware'
```

Comment:
If that works, this is fine.  But the way that I implemented this in one of my GitHub repos was to compare `${GITHUB_REPOSITORY}` to the org/repo name (e.g. `TriBITSPub/TriBITS` in my case).